pkg/testinteraction/common.go

```diff
@@ -0,0 +1,74 @@
+package testinteraction
+
+import (
+	"fmt"
+	"net/http"
+	"regexp"
+
+	"github.com/drud/ddev/pkg/exec"
+	"github.com/headzoo/surf/browser"
+)
+
+// commonInteraction holds values that all interaction structs will need, allowing for common
+// methods to be defined.
+type commonInteraction struct {
+	adminUsername string
```

Comment:
I'm not all the way through this yet, but is it important that it be an adminUsername or just a provided and valid user/pass?